.github/workflows/ci_cd.yml

```diff
@@ -60,7 +60,7 @@ jobs:
       matrix:
         include:
           - python-version: '3.7'
-            vtk-version: '8.1.2'
+            vtk-version: '9.0.3'
           - python-version: '3.8'
             vtk-version: '9.0.3'
```

Comment:
Can we bump the VTK version on each Python version so as to test against a different VTK version in each job

Reply:
I have decided to instead align the versions with https://github.com/pyvista/pyvista/pull/4018 by adding in python 3.11 support.  While this doubles up 9.0.3 on 3.7 and 3.8, this is only temporary until 3.7 support is dropped in both packages.

Reply:
Ah, good call, thanks